### PR TITLE
Port some fixes from no_validatesamfile to master. 

### PR DESCRIPTION
**Pull request by tmooney**
This includes #60 and also enables the relative import.

### CODE REVIEW

File: SingleSampleQc.wdl [hunk 1]
```diff
@@ -24,7 +24,7 @@ version 1.0
 ## - [VerifyBamID2](https://github.com/Griffan/VerifyBamID)
 
 # Git URL import
-import "https://raw.githubusercontent.com/genome/qc-analysis-pipeline/master/tasks/Qc.wdl" as QC
+import "tasks/Qc.wdl" as QC
```
Comment (jasonwalker80) [inline SingleSampleQc.wdl:27]: @acoffman Do you recall why we didn't use relative imports in the past? I feel like this was a limitation of Terra/Cromwell and/or Dockstore.

**Review by jasonwalker80 — APPROVED** [review]
I don't recall specifically why we used the direct/fully qualified import. Other than resolving why and/or testing to confirm that the relative path works just as well, this all looks good to me. +1

File: SingleSampleQc.wdl (hunk 1 above)
Comment (jasonwalker80) [reply SingleSampleQc.wdl:27]: I found a thread on Dockstore that discusses when support was added to both Terra and Dockstore to accept relative paths around Mar/April 2020:
https://discuss.dockstore.org/t/relative-import-path-for-terra/3476/3

I'd say this all looks good then.

Comment (tmooney) [reply SingleSampleQc.wdl:27]: Yeah, I switched the other branch to a relative import after they announced support for it.  424e885ce263eadba2bcea50eb8466272bf2915d was the original change, so we've been using it for awhile.

**Pull request merged**